apps/portals/nf/src/pages/OrganizationDetailsPage/OrganizationDetailsPage.tsx

```diff
@@ -29,7 +29,7 @@ const tabConfig: DetailsPageTabConfig[] = [
 ]
 
 function OrganizationDetailsPage() {
-  const { abbreviation } = useGetPortalComponentSearchParams()
```

Comment:
This details page also supports the searchParam `fundingAgency`, so just pass the entire object along